apps/portals/eliteportal/src/pages/StudyDetailsPage/StudyDataTab.tsx

```diff
@@ -10,9 +10,28 @@ import { DetailsPageContent } from '@sage-bionetworks/synapse-portal-framework/c
 import { DetailsPageContextConsumer } from '@sage-bionetworks/synapse-portal-framework/components/DetailsPage/DetailsPageContext'
 import { MarkdownSynapseFromColumnData } from '@sage-bionetworks/synapse-portal-framework/components/DetailsPage/markdown/MarkdownSynapseFromColumnData'
 import { ColumnSingleValueFilterOperator } from '@sage-bionetworks/synapse-types'
-import { QueryWrapperPlotNav } from 'synapse-react-client'
+import { QueryWrapperPlotNav, RssFeedCards } from 'synapse-react-client'
```

Comment:
Avoid importing entire synapse-react-client bundle on this page
```suggestion
import { QueryWrapperPlotNav } from 'synapse-react-client/components/QueryWrapperPlotNav'
import RssFeedCards from 'synapse-react-client/components/RssFeedCards/RssFeedCards'
```